crypto/crypto_test.ts

```diff
@@ -1418,3 +1418,12 @@ Deno.test({
     assert(stdCrypto.subtle.timingSafeEqual(a.buffer, b.buffer));
   },
 });
+
+// This is one of many methods on `crypto` for which we don't have our own
+// implementation, and just pass calls through to the native implementation.
+// This test doesn't cover any cryptographic logic, but just serves to ensure
+// that (at least this one of) the native methods are indeed re-exported, and
+// that they're appropriately bound to use the required reciever.
```

Comment:
```suggestion
/**
 * This is one of many methods of `crypto` for which we don't have our own
 * implementation, and just pass calls through to the native implementation.
 * This test doesn't cover any cryptographic logic but just serves to ensure
 * that (at least this one of) the native methods are indeed re-exported, and
 * that they're appropriately bound to use the required receiver.
 */
```
Fixes nits like spelling and punctuation.

Reply:
Thanks! Applied the suggested change.